int16 signal1
int16 signal2

